VideoApp/VideoApp/API/TwilioVideoAppAPI.m

```diff
@@ -34,8 +34,6 @@
 static NSString *const kTwilioVideoAppAPIRoomNameKey                        = @"roomName";
 static NSString *const kTwilioVideoAppAPIAppEnvironmentKey                  = @"appEnvironment";
 static NSString *const kTwilioVideoAppAPITopologyKey                        = @"topology";
-static NSString *const kTwilioVideoAppAPIRecordParticipantsOnConnectKey     = @"recordParticipantsOnConnect";
```

Comment:
Yeah... About this one... We actually should have the ability to set this.... If you look at https://twilio.slack.com/archives/CNKH2LC2W/p1582817770041900.. We should be exposing this in an advanced setting similar to Android.

Reply:
Oh ok easy to add back.